packages/legacy-workbench/src/css/login.css

```diff
@@ -0,0 +1,5 @@
+#wb-login .fa-user {
+    font-size: 2rem;
```

Comment:
Aren't there styles for the icon size in the FA stylesheet?

Reply:
`fa-2xl` sets the font-size to 2em

Reply:
No. You shouldn't use it now. These changes should happen during the next phase where we are supposed to apply a new theme based on a design system. Eventually.